app/views/projects/project_submit_info.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="page-header">
+  <h1 class="page-title">Project Creation Information</h1>
+</div>
+
+<div class="card">
+  <div class="card-body">
+    In order to be elegible to be judged and entered to win prizes, you must fist submit your project on Dashboard. Before creating an official submission, review the following requirements:
+    <br>
+    <li> You may only submit one application per project.
+    <li> Every member of the project team must be included in the submission.
+    <li> ADD MORE IMPORTANT FACTS HERE...
```

Comment:
I'm going to wait to commit this until we iron out the participant rules.

Reply:
This should be custom copy in the configuration repo.